### ISSUE TITLE
Restore trash link for custom message template groups

### ISSUE THREAD

**Issue by joshfeck** [eventespresso/event-espresso-core#922]
See related #897 
I'm interested in looking at whether it'd be better for accessibility if the trash link should be viewable on hover or should it always be displayed.

**Comment by nerrad** [comment]
Some notes for making accessible:

- WP removes the `title` attribute and adds `aria-label="Trash"` etc in its list tables for the actions.  This would require updating our `EE_Admin_List` parent to do the same which means _all_ our list tables will be affected (could be a good thing)?

**Comment by nerrad** [comment]
After discussing in [slack](https://eventespresso.slack.com/archives/C080L8G68/p1548941699827500) a decision was made to just restore the actions as they were first, and then the a11y improvements can be done in a separate pull because they impact all our list tables.
